keps/sig-network/1860-kube-proxy-IP-node-binding/kep.yaml

```diff
@@ -16,12 +16,11 @@ approvers:
 
 stage: "alpha"
 
-latest-milestone: "v1.21"
+# latest-milestone: "v1.21"
```

Comment:
@Sh4d1 FYI, PRR approval will need to be part of the PR that sets this back to 1.21.